src/sinks/prometheus.rs

```diff
@@ -21,6 +31,9 @@ pub struct PrometheusSinkConfig {
     pub address: SocketAddr,
     #[serde(default = "default_histogram_buckets")]
     pub buckets: Vec<f64>,
+    /// Should be greater than 1 ms to avoid accidentaly causing infinite loop
```

Comment:
If it's that dangerous, maybe we should validate it before creating the sink, and return a runtime error from `PrometheusSinkConfig ::build` if it's outside some safe range.

Reply:
Fixed in a following commit.